src/components/AppsMenu/components/AppItem.jsx

```diff
@@ -30,7 +30,12 @@ export const AppItem = ({ app }) => {
             label={
               <div className={styles['apps-menu-grid-item']}>
                 <AppIcon app={app} key={app.slug} {...stack.get.iconProps()} />
-                <Typography variant="caption" noWrap className="u-mt-half">
+                <Typography
+                  color="textPrimary"
+                  variant="caption"
```

Comment:
i'm not a big fan of using a variant for the size but with a different color, this may be due to the fact that we use more variants than the design system offers... maybe we should talk to Swann about this?

Reply:
Agree